### PR TITLE
feat: allow custom image width

### LINKED ISSUE #259: Custom width on images
### Describe the feature

At the moment, when you add an image in the Templatical email editor, you can only choose the width from a fixed set of presets: "Full width", "300px", "400px", and "500px". It would be great if, alongside these presets, you could also enter a custom width value yourself.

### Use case

The presets don't always match the exact dimensions a layout needs. A custom width gives finer control so images fit the design precisely, instead of being limited to the nearest preset.

### Alternatives considered

Adjusting the padding to scale the images, but in my opinion that's not a proper solution.

### PR DESCRIPTION
**Pull request by DennisSnijder**
## Summary

Image width could only be one of the presets (Full / 300 / 400 / 500). This adds a **Custom** option to the width dropdown that reveals a px input, so images can be sized to any width (#259). It's editor-only — `ImageBlock.width` is already `number | 'full'` and the renderer already emits arbitrary px widths; this just exposes that in the UI.

## Linked issues

Closes #259.

## Type of change

- [x] New feature (non-breaking change that adds functionality)

## Affected packages

- [x] `@templatical/editor`

## Checklist

- [X] Tests added or updated — none; the render path for numeric widths is already covered, and the only new logic is in the (untested) image toolbar
- [x] `pnpm run ci` passes locally
- [x] Docs — no change needed; `width: number | 'full'` is already documented
- [x] i18n keys added (en + de)
- [x] Changeset added

### CODE REVIEW

File: packages/editor/src/components/toolbar/ImageToolbar.vue [hunk 1]
```diff
@@ -37,10 +42,31 @@ const { start: startPulseSrc } = useTimeoutFn(
...
+  return "custom";
+});
+
 function updateField(field: string, value: unknown): void {
   emit("update", { [field]: value } as Partial<ImageBlock>);
 }
 
+function updateWidthMode(value: string): void {
+  if (value === "custom") {
+    const w = props.block.width;
```
Comment (orkhanahmadov) [inline packages/editor/src/components/toolbar/ImageToolbar.vue:61]: Please guard against empty/NaN and floor at the min. Otherwise empty value would be converted to 0px width - invisible image.

**Review by orkhanahmadov — CHANGES_REQUESTED** [review]
Please also add test coverage for changed behavior and empty/NaN protection

**Review by orkhanahmadov — APPROVED** [review]
